src/Azure/AzureStorage.php

```diff
@@ -5,19 +5,35 @@
 use GemeenteAmsterdam\FixxxSchuldhulp\Azure\Config\SASFileReaderConfig;
 use GemeenteAmsterdam\FixxxSchuldhulp\Azure\Config\SASFileWriterConfig;
 use GuzzleHttp\RequestOptions;
+use Http\Discovery\Exception\NotFoundException;
+use Psr\Cache\InvalidArgumentException;
 use Psr\Log\LoggerInterface;
 use Symfony\Component\Cache\Adapter\FilesystemAdapter;
 use Symfony\Component\HttpFoundation\File\UploadedFile;
 use Symfony\Contracts\Cache\ItemInterface;
+use Symfony\Contracts\HttpClient\Exception\ClientExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\RedirectionExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\ServerExceptionInterface as ServerExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
 use Symfony\Contracts\HttpClient\HttpClientInterface;
 use Symfony\Contracts\HttpClient\ResponseInterface;
 
 class AzureStorage implements AzureStorageInterface
 {
     private const CACHE_KEY = 'azure-access-token';
+    private const TOKEN_REQUEST_SCOPE_STORAGE = 'https://storage.azure.com//.default';
```

Comment:
Dit is niet helemaal up-to-date vanuit de examples nog. Goed teken dat we het inderdaad na golive even uit elkaar moeten trekken en dit centraal moeten hebben. Maar voor nu maar ff handmatig :)